publisher/htmlElementsCollector.go

```diff
@@ -59,7 +48,50 @@ func (h *HTMLElements) Sort() {
 	sort.Strings(h.IDs)
 }
 
-type cssClassCollectorWriter struct {
+type htmlElementsCollector struct {
```

Comment:
You don't have to revert this, it makes sense, but more of a comment for the future: It would be good to avoid these type renames as it makes the change set really big and very hard to review.

Reply:
ok, understood